src/docs/cookbook/navigation/navigation-basics.md

```diff
@@ -68,18 +71,15 @@ class SecondScreen extends StatelessWidget {
 
 ## 2. Navigate to the second screen using `Navigator.push`
 
-In order to Navigate to a new screen, we'll need to use the
-[`Navigator.push`](https://docs.flutter.io/flutter/widgets/Navigator/push.html)
-method. The `push` method will add a `Route` to the stack of routes managed by
-the Navigator!
+The [`Navigator.push`](https://docs.flutter.io/flutter/widgets/Navigator/push.html) method 
```

Comment:
The `Navigator.push(Route)` method pushes the route to the top of the stack of routes managed by the Navigator.

(Notice eliminating "will" in the next line.)